medusa/indexers/tvdbv2/tvdbv2_api.py

```diff
@@ -329,9 +332,9 @@ def _parse_images(self, sid):
 
         for image_type, image_count in self._object_to_dict(series_images_count).iteritems():
             try:
-
                 if search_for_image_type and search_for_image_type != image_type:
-                    continue
+                    if image_type == 'poster' and search_for_image_type != 'poster_thumb':
+                        continue
```

Comment:
We need to use posters for 'poster_thumb' (TVDB doesn't have any poster_thumbs).